stdlib/Test/test/runtests.jl

```diff
@@ -162,7 +162,7 @@ let fails = @testset NoThrowTestSet begin
         @test_throws "A test" error("a test")
         @test_throws r"sqrt\([Cc]omplx" sqrt(-1)
         @test_throws str->occursin("a T", str) error("a test")
-        @test_throws ["BoundsError", "acess", "1-element", "at index [2]"] [1][2]
```

Comment:
I think this is intentional to test the "wrong message" part.

Reply:
Hehehe, well, "aquire" is still not correctly spelled (it's "acquire") ;-)

Reply:
I googled it and saw that there was a wikipedia link for it so I was like "ok all good". It was https://en.wiktionary.org/wiki/aquire...